.github/workflows/go.yml

```diff
@@ -0,0 +1,39 @@
+name: Go
```

Comment:
I've pushed already GitHub action of PR and release flow, could you please remove this one?

Reply:
Done